src/components/Asset/Edit/FormEditMetadata.tsx

```diff
@@ -83,23 +86,48 @@ export default function FormEditMetadata({
 
   return (
     <Form>
-      {data.map(
```

Comment:
@EnzoVezzaro How come you decided to remove the mapping?

Reply:
just to have better control over them like we do in publish. For example [here](https://github.com/oceanprotocol/market/blob/feature/issue-1724-graphql-onchain-storage-support/src/components/Publish/Services/index.tsx#L0-L1), in publish every fields is added manually in case we need to do some special logic on a specific field. Also not sure if the mapping breaks the [graphql / on-chain integration](https://github.com/oceanprotocol/market/pull/1794). Sorry If I didn't included the `paymentCollector` field, I lost it.